assets/js/theme/common/carousel.js

```diff
@@ -14,38 +16,68 @@ const showCarouselIfSlidesAnalizedSetup = ($carousel) => {
         analizedSlides.push($slide);
         if ($slides.length === analizedSlides.length) {
             $carousel.addClass('is-visible');
-            setSlideTabindexes($('.slick-slide'));
         }
     };
 };
 
+const arrowAriaLabling = ($arrowLeft, $arrowRight, currentSlide, lastSlide) => {
+    if (lastSlide < 2) return;
+    if ($arrowLeft.length === 0 || $arrowRight.length === 0) return;
```

Comment:
Can we concatenate conditions?

Reply:
Yes, but in this case I would like to have different conditions, because it is more obvious what cases are unsuitable for continuing function